### PR TITLE
Add UI Tests for Groups.

### PR DESCRIPTION
**Pull request by manjotsidhu**
In this Pull Request I have written all the required UI Tests for Groups View -

- [x] GroupDetailsView
- [x] NewGroupView
- [x] UpdateGroupView
- [x] AddAssignmentView
- [x] UpdateAssignmentView
- [x] AssignmentDetailsView
- [x] ProfileView
- [x] EditProfileView

### CODE REVIEW

**Comment by manjotsidhu** [comment]
@iCoder-007 Can you take a look at this [error](https://pipelines.actions.githubusercontent.com/8AkbQ7Z8xxvFw15rUy3RJX0r6rZDIzKaSuXELA7uTCD0qhykH2/_apis/pipelines/1/runs/265/signedlogcontent/3?urlExpires=2021-01-09T17%3A19%3A31.4880906Z&urlSigningMethod=HMACV1&urlSignature=hXGMDiALkbIjCOXwsivWpdlWTp%2FC6PMuuZrKv%2FDp8u8%3D) caused due to ThemeProvider ?

**Comment by manjotsidhu** [comment]
@Nitish145 @tachyons Kindly review it

File: test/ui_tests/groups/add_assignment_view_test.dart [hunk 1]
```diff
@@ -1,5 +1,116 @@
...
+
+      // Finds no elements checkboxes as Elements Restrictions Checkbox is not selected
+      expect(find.byType(Checkbox), findsOneWidget);
+
+      // Finds Save button
+      expect(find.widgetWithText(CVPrimaryButton, 'Create Assignment'),
+          findsOneWidget);
+    });
+
+    testWidgets('on Create Assingment button is Tapped',
```
Comment (Nitish145) [inline test/ui_tests/groups/add_assignment_view_test.dart:75]: typo

File: test/ui_tests/groups/update_assignment_view_test.dart [hunk 2]
```diff
@@ -1,5 +1,119 @@
...
+
+      // Finds no elements checkboxes as Elements Restrictions Checkbox is not selected
+      expect(find.byType(Checkbox), findsOneWidget);
+
+      // Finds Save button
+      expect(find.widgetWithText(CVPrimaryButton, 'Update Assignment'),
+          findsOneWidget);
+    });
+
+    testWidgets('on Update Assingment button is Tapped',
```
Comment (Nitish145) [inline test/ui_tests/groups/update_assignment_view_test.dart:78]: typo

File: test/ui_tests/profile/user_favourites_view_test.dart [hunk 3]
```diff
@@ -58,7 +58,7 @@ void main() {
       verify(mockObserver.didPush(any, any));
     }
 
-    testWidgets('finds Generic MyGroupsView widgets',
+    testWidgets('finds Generic UserFavourtesView widgets',
```
Comment (Nitish145) [inline test/ui_tests/profile/user_favourites_view_test.dart:61]: typo

**Review by Nitish145 — APPROVED** [review]

**Comment by Nitish145** [comment]
Excellent Work 🥇

**Comment by manjotsidhu** [comment]
Typos have been fixed

**Pull request merged**